### PR TITLE
cleanup under developer guide

### PR DESCRIPTION
**Pull request by rohityadavcloud**
----
📚 Documentation preview 📚: https://cloudstack-documentation--404.org.readthedocs.build/en/404/

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

File: source/developersguide/developer_guide.rst [hunk 1]
```diff
@@ -168,122 +183,38 @@ Start jetty with the simulator enabled
...
-``http://localhost:8080/client`` and start experimenting with the UI or
-the API.
-
-Do note that the management server is running in your local machine and
-that DevCloud is used only as a n Hypervisor. You could potentially run
-the management server within DevCloud as well, or memory granted, run
-multiple DevClouds.
+MonkeyBox or `mbx <https://github.com/shapeblue/mbx>`__
+enable VM/appliance-based CloudStack development. It is tested with Ubuntu and
+uses KVM and prebuilt imagesbto deploy QA and dev environments for anybody to
```
Comment (sureshanaparti) [inline source/developersguide/developer_guide.rst:214]:
```suggestion
uses KVM and prebuilt images to deploy QA and dev environments for anybody to
```

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Review by sureshanaparti — APPROVED** [review]
doc changes lgtm

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**